.github/workflows/test.yml

```diff
@@ -79,4 +74,6 @@ jobs:
 
             # Upload coverage reports to Codecov
             - name: Upload coverage to Codecov
-              uses: codecov/codecov-action@v1
+              uses: codecov/codecov-action@v1.0.10
+              with: 
+                yml: ./codecov.yml
```

Comment:
From the [codecov docs](https://github.com/codecov/codecov-action)
> We've removed the yml parameter with the latest release of this action. Please put your custom codecov yaml file at the root of the repo because other locations will no longer be supported in the future.